packages/gestalt-design-tokens/tokens/vr-theme-web-mapping/sema-color.json

```diff
@@ -16,6 +16,12 @@
       "error": {
         "value": "{sema.color.text.error.value}"
       },
+      "hover": {
```

Comment:
@eniomoura Can you confirm these ones exist in classic so they can be overriden?

We should only map existing ones.

Let me know if you run into some blockers trying to access some tokens.

Reply:
fixed, not sure why i had added this